build.zig

```diff
@@ -24,9 +45,8 @@ pub fn build(b: *std.Build) void {
     triangle_exe.linkLibC();
     triangle_exe.linkSystemLibrary("glfw");
 
-    const vk_xml_path = b.option([]const u8, "vulkan-registry", "Override the path to the Vulkan registry") orelse "examples/vk.xml";
-
-    const gen = vkgen.VkGenerateStep.create(b, vk_xml_path, "vk.zig");
+    const example_registry = b.option([]const u8, "example-registry", "Override the path to the Vulkan registry") orelse "examples/vk.xml";
```

Comment:
The help message here should probably be something like this:
```suggestion
    const example_registry = b.option([]const u8, "example-registry", "Override the path to the Vulkan registry used for the examples") orelse "examples/vk.xml";
```